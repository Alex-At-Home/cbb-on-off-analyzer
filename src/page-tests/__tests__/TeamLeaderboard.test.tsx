import React from 'react';
import TeamLeaderboard from '../../pages/index';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe("TeamLeaderboardPage", () => {
  test("TeamLeaderboardPage - should create snapshot", () => {
    const wrapper = shallow(<TeamLeaderboard testMode={true}/>);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
