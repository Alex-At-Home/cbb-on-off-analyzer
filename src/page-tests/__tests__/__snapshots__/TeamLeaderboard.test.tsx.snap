// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TeamLeaderboardPage TeamLeaderboardPage - should create snapshot 1`] = `
<Container
  fluid={false}
>
  <Head>
    <meta
      content="https://server/thumbnails/team_leaderboard_thumbnail.png"
      property="og:image"
    />
    <meta
      content="https://server/thumbnails/team_leaderboard_thumbnail.png"
      name="twitter:image"
    />
  </Head>
  <Row
    className="mt-2"
    noGutters={false}
  >
    <Col
      className="text-center"
      xs={12}
    >
      <LandingPageIcon />
      <h3>
        Build Your Own T25!
      </h3>
    </Col>
  </Row>
  <Row
    className="border-bottom"
    noGutters={false}
  >
    <HeaderBar
      common={
        Object {
          "gender": "Men",
          "year": "2024/25",
        }
      }
      thisPage="team-_leaderboard"
    />
  </Row>
  <Row
    className="mt-3"
    noGutters={false}
  >
    <TeamLeaderboardTable
      dataEvent={
        Object {
          "confs": Array [],
          "lastUpdated": 0,
          "teams": Array [],
        }
      }
      onChangeState={[Function]}
      startingState={Object {}}
    />
  </Row>
  <Footer
    dateOverride={0}
    server="server"
  />
</Container>
`;
